app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -199,7 +199,8 @@ ControllerContainer::InitializeBeam() {
   }
 }
 
-void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex, const std::vector<vrb::Matrix>& jointTransforms)
+void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex, const std::vector<vrb::Matrix>& jointTransforms,
```

Comment:
We can use move semantics here to avoid deep copies of the vector. Just declare the argument as `std::vector<float> jointRadii` here ...